src/web/auth.rs

```diff
@@ -206,8 +206,10 @@ pub fn hkdf_expand_32(info: &[u8], salt: Option<&[u8]>, key: &[u8]) -> ApiResult
 
 /// Helper function for [HMAC](https://tools.ietf.org/html/rfc2104) verification.
 fn verify_hmac(info: &[u8], key: &[u8], expected: &[u8]) -> ApiResult<()> {
-    let mut hmac: Hmac<Sha256> = Hmac::new_varkey(key)?;
-    hmac.input(info);
+    type HmacSha256 = Hmac<Sha256>;
+
+    let mut hmac: HmacSha256 = HmacSha256::new_varkey(key)?;
```

Comment:
Let's kill the HmacSha256 type alias in these 3 places.

With us having to specify the type parameter at the `new_varkey` callsite it's superfluous to do it on the left side.

```suggestion
    let mut hmac = Hmac::<Sha256>::new_varkey(key)?;
```

Reply:
..and I just noticed the hmac docs' examples use the type alias. I really don't see a good reason why though 🤷‍♂️

Reply:
Probably just legibility. I kinda liked the type for that reason, but yeah, better to be explicit.

Reply:
Forgot the other 2 places